.github/dependabot.yml

```diff
@@ -1,20 +1,35 @@
+ # For details on how this file works refer to:
+ #   - https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
 version: 2
 updates:
   # Maintain dependencies for GitHub Actions
+  #  - Check for updates once a week
   #  - Group all updates into a single PR
-  - package-ecosystem: github-actions
-    directory: /
+  - package-ecosystem: "github-actions"
+    directory: "/"
     schedule:
-      interval: weekly
-      day: monday
-      time: 01:00
-      timezone: Canada/Pacific
+      interval: "weekly"
     groups:
       all-actions:
         patterns: [ "*" ]
 
-  # Maintain dependencies for Python packages
-  - package-ecosystem: pip
-    directory: /server
+  # Maintain dependencies for Python Packages
+  - package-ecosystem: "pip"
+    directory: "/aries-mobile-tests"
     schedule:
-      interval: daily
\ No newline at end of file
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+    ignore:
+      - dependency-name: "*"
+        update-types: ["version-update:semver-major"]
+
+  - package-ecosystem: "docker"
+    directory: "/aries-mobile-tests"
+    schedule:
+      interval: "weekly"
+      day: "monday"
+      time: "04:00"
+      timezone: "Canada/Pacific"
+
```

Comment:
@rajpalc7, There is no `/aries-mobile-tests` folder in this project.  It looks like this code as direct copy of your work here; https://github.com/openwallet-foundation/mobile-wallet-test-harness/pull/253/files

Reply:
@WadeBarnes Ready for review now.